FBSimulatorControl.xcodeproj/project.pbxproj

```diff
@@ -3400,6 +3434,10 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = AA7E55C31CFEC5B6009209AE /* FBDeviceControlTests.xcconfig */;
 			buildSettings = {
+				FRAMEWORK_SEARCH_PATHS = (
```

Comment:
If you follow my suggestion about passing the values for macOS and Xcode versions, then we can avoid needing to stub these methods in the tests.